### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by tadanokojin**
Update to Github's new issue template system.
Provides templates for feature requests and bug reports.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+If your issue is graphical in nature or you think screenshots will help illustrate your issue, you may do that here. Otherwise you may delete this section.
+
+**System Info (please complete the following information):**
+ - PCSX2 Revision: [e.g. dev-525]
+ - OS: [e.g. Windows 10]
+ - CPU: [e.g. i5-7600]
+ - GPU: [e.g. GTX 1070]
+
+**Logs and Dumps**
+Please feel free to attach any logs, block dumps, [gsdump](https://forums.pcsx2.net/Thread-How-to-create-a-proper-GS-dump) etc here. If your problem is graphical in nature it is highly recommended that you provide a gsdump.
```
Comment (RedDevilus) [inline .github/ISSUE_TEMPLATE/bug_report.md:43]: Maybe include the mention of uploading savestates / memcard files or even? To flesh this more out a little bit. Gsdump doesn't always help on a graphical issue or what people think of a gs problem.

Comment (tadanokojin) [reply .github/ISSUE_TEMPLATE/bug_report.md:43]: If you can provide links to those things on the forums or whatever I'll include them. But I don't really want this template to be a guide.

Comment (RedDevilus) [reply .github/ISSUE_TEMPLATE/bug_report.md:43]: Fair enough, i don't seem to find anything on the forum except what's already in the PDF guides that are included with PCSX2 or some trial and error. Should be fine with this though.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+Notes:
+- Please do not make support requests on Github. Our issue tracker is for tracking bugs and feature requests only. If you need help configuring the emulator please make a [request on our forums](https://forums.pcsx2.net/index.php) or contact us on [discord](https://discord.com/invite/TCz3t9k). If you are unsure, start with discord or the forums.
+- We only accept bug reports for the latest dev version. Please try upgrading before making an issue.
+- Please make an effort to make sure your issue isn't already reported.
+- We are not accepting bug reports for PSX mode at this time. If you are interested in helping contribute to PSX mode please do so on the [forums](https://forums.pcsx2.net/index.php). Otherwise our recommendation is that you use a proper [PSX emulator](https://emulation.gametechwiki.com/index.php/PlayStation_emulators).
```
Comment (xTVaser) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: Generally my issue with templates is always that they put a pile of placeholder text that ends up on the final issue and makes it harder to read.  I'd lean towards putting any exposition in comments so it doesn't end up on the final issue regardless of whether-or-not the creator cleans things up. For example - https://github.com/aws/aws-sdk-java/issues/new?labels=bug%2C+needs-triage&template=bug-report.md

**Review by xTVaser — CHANGES_REQUESTED** [review]
hmm, I've seen this issue template system used in other repos and its really nice.  But not sure what the templates you made look like in action (ie. maybe you use things the other repos I've seen don't).  I guess i could throw them onto my fork's master to see.  But even if there are some slight issues that appear once merged, no doubt this is an improvement.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (tadanokojin) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: Good call.

**Comment by tadanokojin** [comment]
Yeah feel free to try it out.

**Review by lightningterror — APPROVED** [review]

**Comment by guoyunhe** [comment]
This PR blocked issue creation. If you click the new issue button, you will see three links:

图片

None of them allow you to create issue. You have go to https://github.com/PCSX2/pcsx2/issues/new manually to create an issue. This is not friendly for users.

**Pull request merged**